spec/hdinsight_cluster_service_spec.rb

```diff
@@ -0,0 +1,63 @@
+#############################################################################
+# hdinsight_cluster_service_spec.rb
+#
+# Test suite for the Azure::Armrest::HDInsight::HDInsightClusterService class.
+#############################################################################
+require 'spec_helper'
+
+describe "HDInsight::HDInsightClusterService" do
+  before { setup_params }
+  let(:server) { Azure::Armrest::HDInsight::HDInsightClusterService.new(@conf) }
+
+  context "inheritance" do
+    it "is a subclass of ArmrestService" do
+      expect(Azure::Armrest::HDInsight::HDInsightClusterService.ancestors).to include(Azure::Armrest::ArmrestService)
+    end
+  end
+
+  context "constructor" do
+    it "returns a HDInsightCluster instance as expected" do
+      expect(server).to be_kind_of(Azure::Armrest::HDInsight::HDInsightClusterService)
+    end
+  end
+
+  context "accessors" do
```

Comment:
I am thinking we only need the first two tests. All the rest tests are from base class, which are already tested. You can delete them all. Same for the other test.

Reply:
I don't see a spec for the subservice class, are those tests all done in the base service class? Also, yes, I shamelessly ripped off the SQL database and SQL server specs